base/applications/regedit/lang/pt-PT.rc

```diff
@@ -103,35 +103,35 @@ BEGIN
         BEGIN
             MENUITEM "&Chave", ID_EDIT_NEW_KEY
             MENUITEM SEPARATOR
-            MENUITEM "Valor da &cadeia", ID_EDIT_NEW_STRINGVALUE
-            MENUITEM "Valor &Binário", ID_EDIT_NEW_BINARYVALUE
+            MENUITEM "&Valor da cadeia", ID_EDIT_NEW_STRINGVALUE
+            MENUITEM "Valor &binário", ID_EDIT_NEW_BINARYVALUE
             MENUITEM "Valor &DWORD", ID_EDIT_NEW_DWORDVALUE
-            MENUITEM "Valor &Multi-cadeia", ID_EDIT_NEW_MULTISTRINGVALUE
-            MENUITEM "Valor c&adeia expansível", ID_EDIT_NEW_EXPANDABLESTRINGVALUE
+            MENUITEM "Valor &multi-cadeia", ID_EDIT_NEW_MULTISTRINGVALUE
+            MENUITEM "Valor de cadeia e&xpansível", ID_EDIT_NEW_EXPANDABLESTRINGVALUE
         END
         MENUITEM SEPARATOR
         MENUITEM "&Permissões...", ID_EDIT_PERMISSIONS
         MENUITEM SEPARATOR
-        MENUITEM "&Excluir\tDel", ID_EDIT_DELETE
-        MENUITEM "&Mudar o nome", ID_EDIT_RENAME
+        MENUITEM "Elimina&r\tDel", ID_EDIT_DELETE
```

Comment:
The file has 2x
```
        MENUITEM "&Excluir\tDel", ID_EDIT_DELETE
        MENUITEM "&Excluir\tDel", ID_EDIT_DELETE

```and one time
        `MENUITEM "&Excluir", ID_HEXEDIT_DELETE`
and uses the word Excluir also in a few other strings.
When using` "Elimina&r\tDel"` here now, all of that gets rather inconsisten. I therefore recommend to stick with the old `"&Excluir\tDel"` because that didn't even collide with anything.
```suggestion
        MENUITEM "&Excluir\tDel", ID_EDIT_DELETE
```